### ISSUE TITLE
Expand installation instructions

### ISSUE THREAD

**Issue by hagenw** [soundata/soundata#153]
When running `pip install soundata` in a virtual environment can fail, e.g.

```bash
$ virtualenv --python="python3.10"  venv
$ source venv/bin/activate
$ pip install soundata
...
Building wheels for collected packages: simpleaudio                                                                                                                             Building wheel for simpleaudio (setup.py) ... error                                  
  error: subprocess-exited-with-error                                                  
                                                                                                                                                                              
  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1                                                                       
  ╰─> [37 lines of output]                                                             
      running bdist_wheel                                                              
      running build                                                                    
      running build_py                                                                                                                                                        
      creating build                                                                                                                                                          
      creating build/lib.linux-x86_64-3.10                                                                                                                                    
      creating build/lib.linux-x86_64-3.10/simpleaudio                                                                                                                              copying simpleaudio/__init__.py -> build/lib.linux-x86_64-3.10/simpleaudio                                                                                                    copying simpleaudio/shiny.py -> build/lib.linux-x86_64-3.10/simpleaudio                                                                                                 
      copying simpleaudio/functionchecks.py -> build/lib.linux-x86_64-3.10/simpleaudio                                                                                        
      creating build/lib.linux-x86_64-3.10/simpleaudio/test_audio
      copying simpleaudio/test_audio/c.wav -> build/lib.linux-x86_64-3.10/simpleaudio/test_audio
      copying simpleaudio/test_audio/e.wav -> build/lib.linux-x86_64-3.10/simpleaudio/test_audio
      copying simpleaudio/test_audio/g.wav -> build/lib.linux-x86_64-3.10/simpleaudio/test_audio
      copying simpleaudio/test_audio/left_right.wav -> build/lib.linux-x86_64-3.10/simpleaudio/test_audio
      copying simpleaudio/test_audio/notes_2_16_44.wav -> build/lib.linux-x86_64-3.10/simpleaudio/test_audio
      running build_ext                                                                                                                                                       
      building 'simpleaudio._simpleaudio' extension
      creating build/temp.linux-x86_64-3.10
      creating build/temp.linux-x86_64-3.10/c_src                                      
      x86_64-linux-gnu-gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -g -fst
ack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -DDEBUG=0 -I/home/hwierstorf/.envs/test/include -I/usr/include/python3.10 -c c_src
/posix_mutex.c -o build/temp.linux-x86_64-3.10/c_src/posix_mutex.o                     
      x86_64-linux-gnu-gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -g -fst
ack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -DDEBUG=0 -I/home/hwierstorf/.envs/test/include -I/usr/include/python3.10 -c c_src
/simpleaudio.c -o build/temp.linux-x86_64-3.10/c_src/simpleaudio.o           
      c_src/simpleaudio.c: In function ‘_play_buffer’:                                
      c_src/simpleaudio.c:164:5: warning: ‘PyEval_InitThreads’ is deprecated [-Wdeprecated-declarations]
        164 |     PyEval_InitThreads();
            |     ^~~~~~~~~~~~~~~~~~
      In file included from /usr/include/python3.10/Python.h:130,
                       from c_src/simpleaudio.h:10,
                       from c_src/simpleaudio.c:7:
      /usr/include/python3.10/ceval.h:122:37: note: declared here
        122 | Py_DEPRECATED(3.9) PyAPI_FUNC(void) PyEval_InitThreads(void);
            |                                     ^~~~~~~~~~~~~~~~~~
      x86_64-linux-gnu-gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -g -fst
ack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -DDEBUG=0 -I/home/hwierstorf/.envs/test/include -I/usr/include/python3.10 -c c_src
/simpleaudio_alsa.c -o build/temp.linux-x86_64-3.10/c_src/simpleaudio_alsa.o
      c_src/simpleaudio_alsa.c:8:10: fatal error: alsa/asoundlib.h: No such file or directory
          8 | #include <alsa/asoundlib.h>
            |          ^~~~~~~~~~~~~~~~~~
      compilation terminated.
      error: command '/usr/bin/x86_64-linux-gnu-gcc' failed with exit code 1
      [end of output]
   
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for simpleaudio
  Running setup.py clean for simpleaudio
Failed to build simpleaudio
Installing collected packages: wcwidth, texttable, sortedcontainers, simpleaudio, pytz, pydub, pure-eval, ptyprocess, brotli, widgetsnbextension, urllib3, tzdata, typing-exte
nsions, traitlets, tqdm, tomli, threadpoolctl, six, rpds-py, pyzstd, pyppmd, pyparsing, pygments, pyflakes, pycryptodomex, pycparser, pycodestyle, pybcj, psutil, prompt-toolk
it, platformdirs, pillow, pexpect, parso, packaging, numpy, mypy-extensions, multivolumefile, msgpack, mccabe, llvmlite, lazy-loader, kiwisolver, jupyterlab-widgets, joblib, 
inflate64, idna, future, fonttools, executing, exceptiongroup, decorator, cycler, charset-normalizer, certifi, audioread, attrs, soxr, scipy, requests, referencing, python-da
teutil, py7zr, numba, mypy, matplotlib-inline, jedi, flake8, contourpy, comm, cffi, asttokens, stack-data, soundfile, scikit-learn, pooch, pandas, mir-eval, matplotlib, jsons
chema-specifications, seaborn, librosa, jsonschema, ipython, jams, ipywidgets, soundata 
  Running setup.py install for simpleaudio ... error
  error: subprocess-exited-with-error
   
  × Running setup.py install for simpleaudio did not run successfully.
  │ exit code: 1
  ╰─> [39 lines of output]
      running install
      /home/hwierstorf/.envs/test/lib/python3.10/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and p
ip and other standards-based tools.
        warnings.warn(
      running build
      running build_py
      creating build
      creating build/lib.linux-x86_64-3.10
      creating build/lib.linux-x86_64-3.10/simpleaudio
      copying simpleaudio/__init__.py -> build/lib.linux-x86_64-3.10/simpleaudio
      copying simpleaudio/shiny.py -> build/lib.linux-x86_64-3.10/simpleaudio
      copying simpleaudio/functionchecks.py -> build/lib.linux-x86_64-3.10/simpleaudio
      creating build/lib.linux-x86_64-3.10/simpleaudio/test_audio
      copying simpleaudio/test_audio/c.wav -> build/lib.linux-x86_64-3.10/simpleaudio/test_audio
      copying simpleaudio/test_audio/e.wav -> build/lib.linux-x86_64-3.10/simpleaudio/test_audio
      copying simpleaudio/test_audio/g.wav -> build/lib.linux-x86_64-3.10/simpleaudio/test_audio
      copying simpleaudio/test_audio/left_right.wav -> build/lib.linux-x86_64-3.10/simpleaudio/test_audio
      copying simpleaudio/test_audio/notes_2_16_44.wav -> build/lib.linux-x86_64-3.10/simpleaudio/test_audio
      running build_ext
      building 'simpleaudio._simpleaudio' extension
      creating build/temp.linux-x86_64-3.10
      creating build/temp.linux-x86_64-3.10/c_src
      x86_64-linux-gnu-gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -g -fst
ack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -DDEBUG=0 -I/home/hwierstorf/.envs/test/include -I/usr/include/python3.10 -c c_src
/posix_mutex.c -o build/temp.linux-x86_64-3.10/c_src/posix_mutex.o
      x86_64-linux-gnu-gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -g -fst
ack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -DDEBUG=0 -I/home/hwierstorf/.envs/test/include -I/usr/include/python3.10 -c c_src
/simpleaudio.c -o build/temp.linux-x86_64-3.10/c_src/simpleaudio.o
      c_src/simpleaudio.c: In function ‘_play_buffer’:
      c_src/simpleaudio.c:164:5: warning: ‘PyEval_InitThreads’ is deprecated [-Wdeprecated-declarations]
        164 |     PyEval_InitThreads();
            |     ^~~~~~~~~~~~~~~~~~
      In file included from /usr/include/python3.10/Python.h:130,
                       from c_src/simpleaudio.h:10,
                       from c_src/simpleaudio.c:7:
      /usr/include/python3.10/ceval.h:122:37: note: declared here
        122 | Py_DEPRECATED(3.9) PyAPI_FUNC(void) PyEval_InitThreads(void);
            |                                     ^~~~~~~~~~~~~~~~~~
      x86_64-linux-gnu-gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -g -fst
ack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -DDEBUG=0 -I/home/hwierstorf/.envs/test/include -I/usr/include/python3.10 -c c_src
/simpleaudio_alsa.c -o build/temp.linux-x86_64-3.10/c_src/simpleaudio_alsa.o
      c_src/simpleaudio_alsa.c:8:10: fatal error: alsa/asoundlib.h: No such file or directory
          8 | #include <alsa/asoundlib.h>
            |          ^~~~~~~~~~~~~~~~~~
      compilation terminated.
      error: command '/usr/bin/x86_64-linux-gnu-gcc' failed with exit code 1
      [end of output]
   
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: legacy-install-failure

× Encountered error while trying to install package.
╰─> simpleaudio

note: This is an issue with the package mentioned above, not pip.
hint: See above for output from the failure.
```

As stated at https://simpleaudio.readthedocs.io/en/latest/installation.html this can be solved under Ubunbtu/Debian by running

```bash
$ sudo apt-get install -y libasound2-dev
```

Would be nice to add a short statement to the installation/contributing instructions that this is needed. Or link to the corresponding `simpleaudio` instructions.

**Comment by hadware** [comment]
I agree with this. This is also linked to https://github.com/soundata/soundata/issues/154 and should be solved.

**Comment by guillemcortes** [comment]
Closing this issue since it has been addressed in #166 . Many thanks!
